examples/poem-openapi/README.md

```diff
@@ -0,0 +1,24 @@
+# poem-openapi with passage_flex
+
+This is an example of how to use the `passage_flex` crate with an HTTP server implemented with [poem-openapi](https://crates.io/crates/poem-openapi) in Rust.
+
+
+## How to run
+
+Set `PASSAGE_APP_ID` and `PASSAGE_API_KEY` environment variables with valid values:
+```bash
+export PASSAGE_APP_ID=your-app-id
+export PASSAGE_API_KEY=your-api-key
```

Comment:
how much more effort would it be to place these in a `.env` file instead? just wary that someone who is following these commands is leaking their api key in the terminal.

it's unlikely that it will be an actual issue, but if the change is easy enough then it might be nice to follow best practices since this is a public example.

Reply:
to circle back, inline env vars should suffice to keep other processes from seeing it, but an env file isn't necessary since a downside to it is that a dev could leave that on their machine.